config/SOUE01/rels/d_a_obj_rail_postNP/symbols.txt

```diff
@@ -4,15 +4,15 @@ _unresolved = .text:0x00000060; // type:function size:0x4 scope:global
 dAcORailPost_classInit__Fv = .text:0x00000070; // type:function size:0x30
 __ct__12dAcORailPostFv = .text:0x000000A0; // type:function size:0x4C
 __dt__12dAcORailPostFv = .text:0x000000F0; // type:function size:0x74
-AcORailPost__initModels = .text:0x00000170; // type:function size:0x118
-AcORailPost__init = .text:0x00000290; // type:function size:0xA4
-AcORailPost__destroy = .text:0x00000340; // type:function size:0x38
-fn_524_380 = .text:0x00000380; // type:function size:0x8
-AcORailPost__draw = .text:0x00000390; // type:function size:0x34
+createHeap__12dAcORailPostFv = .text:0x00000170; // type:function size:0x118
+create__12dAcORailPostFv = .text:0x00000290; // type:function size:0xA4
+doDelete__12dAcORailPostFv = .text:0x00000340; // type:function size:0x38
+actorExecute__12dAcORailPostFv = .text:0x00000380; // type:function size:0x8
+draw__12dAcORailPostFv = .text:0x00000390; // type:function size:0x34
 _ctors = .ctors:0x00000000; // type:label scope:global
 _dtors = .dtors:0x00000000; // type:label scope:global
-lbl_524_rodata_0 = .rodata:0x00000000; // type:object size:0x4 data:float
+@15367 = .rodata:0x00000000; // type:object size:0x4 data:float
 g_profile_OBJ_RAIL_POST = .data:0x00000000; // type:object size:0x10
-lbl_524_data_10 = .data:0x00000010; // type:object size:0xC
-lbl_524_data_1C = .data:0x0000001C; // type:object size:0x1C
-AcORailPost__vtable = .data:0x00000038; // type:object size:0x80
+@15350 = .data:0x00000010; // type:object size:0xC
+@15366 = .data:0x0000001C; // type:object size:0x1C
+__vt__12dAcORailPost = .data:0x00000038; // type:object size:0x80
```

Comment:
instead of renaming the symbols with `@` follow https://wiki.decomp.dev/en/projects/gamecube-wii/skyward-sword#how-to-deal-with-local-anonymous-symbols and replace with adding `scope:local` to the end.

Reply:
Does the newest commit fix it?